dj_audit/middleware.py

```diff
@@ -117,7 +118,7 @@ def process_response(self, request, response=None):
                     else:
                         log_type = 'failed'
             if response_type == 'http':
-                response_body = ''
+                response_body = 'html content'
```

Comment:
@gemdajs do we have to hard code the text 'html content' ?

Reply:
Not really, just needed to ensure the value passes a test a few lines down the source code.